UICollectionViewEmptyState/UICollectionView+EmptyState.m

```diff
@@ -174,7 +174,12 @@ - (void) __empty_layoutHeader {
   }
 
   // always update frame
-  CGRect rect = UIEdgeInsetsInsetRect(bounds, self.contentInset);
+  CGRect rect;
+  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion >= 11) {
```

Comment:
I'm not against upgrading the project requirements, but since this is ObjC, you can safely check for availability here with `respondsToSelector()`. That would be better, in my opinion